internal/deploy/kustomize/daily/overlays/watch-all-namespaces/wlo-all-namespaces.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: websphere-liberty-controller-manager
+  namespace: websphere-liberty
```

Comment:
This shouldn't be necessary as the kustomization.yaml file should add in the namespace.